SETUP/upgrade/13/20191228_alter_past_tallies.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Dropping redundant key on past_tallies...\n";
+$sql = "
+    ALTER TABLE past_tallies
+        DROP INDEX tallyboard_time;
```

Comment:
`explain select holder_id, tally_value from past_tallies where timestamp = 1577509200 and tally_name='P1' and holder_type = 'U';` says that it uses the `tallyboard_time` index.

Reply:
Yup, thanks. I didn't see that one come through when I looked through the query logs. I'll drop the change on this table altogether.